stock_card_report/reports/stock_card_report.xml

```diff
@@ -0,0 +1,154 @@
+<odoo>
+    <template id='assets_backend'
+              name='stock_card_report assets' inherit_id='web.assets_backend'>
+        <xpath expr='.' position='inside'>
+            <link href='/stock_card_report/static/src/css/report.css' rel='stylesheet'/>
+            <script type='text/javascript'
+                    src='/stock_card_report/static/src/js/stock_card_report_backend.js'/>
+            <script type='text/javascript'
+                    src='/stock_card_report/static/src/js/stock_card_report_widgets.js'/>
+        </xpath>
+    </template>
+
+    <template id='stock_card_report.internal_layout'>
+        <div class='article'>
+            <link href='/stock_card_report/static/src/css/report.css' rel='stylesheet'/>
+          <t t-raw='0' />
+        </div>
+        <div class='footer'>
+            <div class='row'>
+                <div class='col-6 custom_footer'>
+                    <span t-esc="context_timestamp(datetime.datetime.now()).strftime('%Y-%m-%d %H:%M')"/>
+                </div>
+                <div class='col-6 text-right custom_footer'>
+                    <span class='page'/>/<span class='topage'/>
+                </div>
+            </div>
+        </div>
+    </template>
+
+    <template id='report_buttons'>
+        <div class='button_row'>
+            <button class='o_stock_card_reports_print btn btn-sm oe_button'><span class='fa fa-print'/>Print</button>
+            <button class='o_stock_card_reports_export btn btn-sm oe_button'><span class='fa fa-download'/>Export</button>
+        </div>
+    </template>
+
+    <template id='report_stock_card_report'>
+        <div class='o_stock_card_reports_page'>
+            <t t-call='stock_card_report.report_buttons'/>
+            <t t-call='stock_card_report.report_stock_card_report_base'/>
+        </div>
+    </template>
+
+    <template id='stock_card_report.report_stock_card_report_pdf'>
```

Comment:
I got this error when try to run report,
```
load could not load template
Template: stock_card_report.report_stock_card_report_base
```
May be this is reference before it is declared?